packages/visualizations/src/components/Table/store.ts

```diff
@@ -1,17 +1,7 @@
 // eslint-disable-next-line import/no-extraneous-dependencies
 import { writable } from 'svelte/store';
 
-type TableStore = {
-    locale: string;
-};
-
 const defaultLocale = navigator.language;
 
-const store = writable<TableStore>({ locale: defaultLocale });
-
-const updateLocale = (locale: string = defaultLocale) => {
-    store.update((previousValue) => ({ ...previousValue, locale }));
-};
-
-export { updateLocale };
-export default store;
+// eslint-disable-next-line import/prefer-default-export
+export const locale = writable<string>(defaultLocale);
```

Comment:
The `locale` prop doesn't seems to be reactive anymore. 

I can update once the locale and see the change, but after that, any change of value for the locale no longer change the format (dates for example)

You can check on the Storybook of the main branch to see the difference with this branch 

Adding a react test, might prevent us to break this in future update. If I remember well, it's the second time we encountered this issue.

Reply:
Ok, I looked into it and it's not trivial at all and not directly related to the store. The $locale is still reactive, the problem arrise when we call `display` (even with a fixed locale actually). I suspect this is because we have nested functions, one being a prop, but I can't seem to find my way around it for now… 🤔

Reply:
Oh ok, so it might not be related to Svelte but Storybook. 
When some props are functions, Storybook stories might be buggy if we update some props

Could we try to add React tests to asset that both locale and display work as expected ?

Reply:
So I just found the reason behind our bug

The display function is not a dependency for the computed `value` as we did it for the locale prop.

So when `display` changes, value doesn't change. 

We should either pass display as an argument of the format function or declare the `format` function via a computation